src/EnumSet.php

```diff
@@ -54,19 +53,12 @@ public function __construct($enumeration)
                 __NAMESPACE__ . '\Enum'
             ));
         }
-
+        
         $this->enumeration = $enumeration;
         $this->ordinalMax  = count($enumeration::getConstants());
-
-        if (PHP_INT_SIZE * 8 < $this->ordinalMax) {
-            throw new OutOfRangeException(sprintf(
-                "Your system can handle up to %u enumerators within an EnumSet"
-                . " but the given enumeration '%s' has defined %u enumerators",
-                PHP_INT_SIZE * 8,
-                $enumeration,
-                $this->ordinalMax
-            ));
-        }
+        
+        //init the bitset with zeros
+        $this->setBitset("");
```

Comment:
Micro optimization: `$this->bitset = str_repeat("\0", ceil($this->ordinalMax / 8));`